src/autocomplete/index.ts

```diff
@@ -81,7 +81,9 @@ class Autocomplete {
     }
     if (activate) {
       this.state.selected = index;
-      this.updateProducts(target);
+      if (index !== -1) {
```

Comment:
we should save `index !== -1` as a `const` at the top of this method since we now use it twice